README.md

```diff
@@ -4,22 +4,65 @@ A service which helps you integrate your upstream GitHub project into Fedora
 ecosystem. Think of it as [packit](https://github.com/packit-service/packit) on
 steroids.
 
+## Introductory description
+With packit, it's easy to integrate your upstream project into Fedora Project.
+Your pull requests are built in Fedora version of your choice,
+so you're always sure that your software works. When you're ready for a release,
+packit takes care of all the hassle and is able to ship your new upstream release
+into Fedora Rawhide.
+
+### Easy to use
+
+Using Packit Service is very straightforward: add one config file to your repository,
+followed by an RPM spec file and you're good to go. We also support packit as a CLI tool,
+so you can always try things locally on your own.
+
+### Makes you confident
+
+Packit Service validates your pull requests by building your software in Fedora OS.
+Once the builds are done, packit lets you know how to install your change
+inside your environment. Get confidence by trying out your changes yourself
+before shipping them to your users.
+
+## Steps how to install Packit-as-a-Service into your projects or organizations
```

Comment:
I'm still in favour having this on our site (packit.dev/docs) instead.

README of this repository should inform people how to develop packit service and not serve as a documentation for users.

Reply:
Yeah, it should be also on `packit.dev/docs` or in `docs.packit.dev`. That's true.
But once users look at this repository, they will ask, how `Packit-as-a-Service` can be installed on their repo.
I guess both sides, as `packit.dev/docs` as this repository should contain this information.

That's my thinking from the user point of view.

Reply:
> Yeah, it should be also on `packit.dev/docs` or in `docs.packit.dev`. That's true.
> But once users look at this repository, they will ask, how `Packit-as-a-Service` can be installed on their repo.
> I guess both sides, as `packit.dev/docs` as this repository should contain this information.
> 
> That's my thinking from the user point of view.

I disagree.

The workflow you describe is usually true for tools, but not for services. Example: how many times have you looked at github repo of travis CI? Did you look for documentation in there? I personally always access travis' web interface and look for documentation in there. Therefore I'd say we should navigate people to our website when it comes to service, not to our repositories.

Reply:
Travis CI example is really relevant here, once our app will be in the marketplace, there is no reason for users who just wants to use packit as a service to search for documentation in our GitHub repository where a lot of information is not important for them, so I am more inclined towards having it just on our website.

Reply:
OK, then I would prefer to have here only the first section, because of we don't have `docs` page and once we have it, either move this part into `docs` page if GitHub App will no be in the marketplace or place in the `docs` second paragraph.

WDYT?

Reply:
@TomasTomecek @rpitonak What do you think about my last comment? Or remove completely section `## Steps how to install...`

Reply:
I think the section can be there and should just point to our website.

Reply:
Good point. I will create a Pull Request to packit.dev repository.